randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3107,6 +3113,13 @@ Extra - asset_id: collision_camera_038
           All of the following:
               Heat/Cold Runs (Advanced) and Lava Damage ≥ 500
               Varia Suit or Heat Damage ≥ 500
+  > Event - Lower Lava Button
+      All of the following:
+          Shoot Diffusion or Wave
+          Any of the following:
+              Gravity Suit
+              Varia Suit and Heat/Cold Runs (Intermediate) and Lava Damage ≥ 200
+              Flash Shift and Heat/Cold Runs (Advanced) and Heat Damage ≥ 425 and Lava Damage ≥ 425
```

Comment:
Like the earlier case, gonna need to add a new event node. These event nodes you will be connecting back to the nodes you connected it to.

Reply:
I did move the connection to rely on pre-existing damage run tricks to get to that point, hopefully it was implemented correctly